supervisord.log

```diff
@@ -13,3 +13,60 @@
 2020-06-12 16:23:51,305 INFO success: php-fpmd entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
```

Comment:
add this file to gitignore and remove it from the repo

Reply:
Done